src/uct/api/v2/uct_v2.h

```diff
@@ -121,6 +121,78 @@ typedef struct {
 } uct_perf_attr_t;
 
 
+/**
+ * @ingroup UCT_MD
+ * @brief MD memory de-registration operation flags.
+ */
+typedef enum {
+    UCT_MD_MEM_DEREG_FIELD_MEMH     = UCS_BIT(0), /**< memh field */
+    UCT_MD_MEM_DEREG_FIELD_FLAGS    = UCS_BIT(1), /**< flags field */
+    UCT_MD_MEM_DEREG_FIELD_CALLBACK = UCS_BIT(2), /**< cb field */
+    UCT_MD_MEM_DEREG_FIELD_ARG      = UCS_BIT(3)  /**< arg field */
+} uct_md_mem_dereg_field_mask_t;
+
+
+typedef enum {
+    /**
+     * Invalidate memory region. If this flag is set then memory region is
+     * invalidated after de-registration and callback (see @ref
+     * uct_md_mem_dereg_params_t) is is called when the memory is fully
+     * invalidated and would not be accessed anymore by zero-copy or remote
+     * memory access operations.
+     */
+    UCT_MD_MEM_DEREG_FLAG_INVALIDATE = UCS_BIT(0) 
+} uct_md_mem_dereg_flags_t;
+
+
+/**
+ * @ingroup UCT_MD
+ * @brief Completion callback for memory region invalidation.
+ *
+ * This callback routine is invoked when when it is no longer accessible by
```

Comment:
`it is no longer accessible`  - ? `memory is no longer remotely accessible` ? Does it mean that it is invoked post de-registeration call